libraries/rush-lib/src/logic/operations/CacheableOperationPlugin.ts

```diff
@@ -99,6 +99,25 @@ export class CacheableOperationPlugin implements IPhasedCommandPlugin {
           ? new DisjointSet()
           : undefined;
 
+        for (const operation of recordByOperation.keys()) {
+          if (
+            operation.settings?.allowCobuildOrchestration &&
+            !cobuildConfiguration?.cobuildOrchestrationOnlyAllowed
+          ) {
+            throw new Error(
+              `Operation ${operation.name} is not allowed to run without the cobuild orchestration experiment enabled. You must use the RUSH_COBUILD_ORCHESTRATION_ONLY_ALLOWED environment variable to enable that mode.`
```

Comment:
Reference the environment variable name const.

Reply:
done in [c0fd113](https://github.com/microsoft/rushstack/pull/4871/commits/c0fd113fc40764c6525cec84d7ac06eeae836b68)